src/placement/pl_map_common.c

```diff
@@ -63,6 +63,50 @@ remap_add_one(d_list_t *remap_list, struct failed_shard *f_new)
 	d_list_add(&f_new->fs_list, remap_list);
 }
 
+void
+reint_add_one(d_list_t *remap_list, struct failed_shard *f_new)
+{
+	struct failed_shard     *f_shard;
+	d_list_t                *tmp;
+
+	D_DEBUG(DB_PL, "Reint target : fnew: %u", f_new->fs_shard_idx);
+
+	/* All failed shards are sorted by fseq in ascending order */
```

Comment:
Looks it's not in fseq order, and I don't think it's necessary in fseq order.

Reply:
Correct, this is important as part of rebuild and finding fallback shards but is unnecessary as part of reintegration.

Will remove.